packages/datasource/src/api/common.ts

```diff
@@ -0,0 +1,3 @@
+import client from './client';
+
+export const searchCompany = (params: { word: string }) => client.get('/api/search-company?word=' + params.word);
```

Comment:
```js
client.get('...', { params: { word }})
```
see [axios document](https://github.com/axios/axios)

If you don't use `param` parameter, make sure you have encoded url search params' key and value to prevent illegal characters like `&` `=` `#`

Reply:
OK